config/index.js

```diff
@@ -47,7 +47,8 @@ const config = {
       accessToken: process.env.ROLLBAR_ACCESS_TOKEN,
       environment: process.env.ROLLBAR_ENV
     },
-    port: process.env.PORT
+    port: process.env.PORT,
+    albumsUrl: process.env.ALBUMS_URL
```

Comment:
You should put this in `config/testing.js` too, with a default value for tests. And you should add the env var in travis :see_no_evil:

Reply:
As i understand, this `common` fields are merge with the ones that are specific to the current environment.

```js
const configFile = `./${ENVIRONMENT}`;
...
const customConfig = require(configFile).config;
module.exports = assignObject(customConfig, config);
```